pylate/hf_hub/model_card.py

```diff
@@ -231,25 +228,21 @@ def to_dict(self) -> dict[str, Any]:
             self.model.get_sentence_embedding_dimension()
         )
         super_dict["model_string"] = str(self.model)
+
         if self.model.similarity_fn_name:
-            super_dict["similarity_fn_name"] = {
-                "cosine": "Cosine Similarity",
-                "dot": "Dot Product",
-                "euclidean": "Euclidean Distance",
-                "manhattan": "Manhattan Distance",
-            }.get(
-                self.model.similarity_fn_name,
-                self.model.similarity_fn_name.replace("_", " ").title(),
-            )
+            super_dict["similarity_fn_name"] = self.model.similarity_fn_name
         else:
-            super_dict["similarity_fn_name"] = "Cosine Similarity"
-
+            super_dict["similarity_fn_name"] = "MaxSim"
         self.first_save = False
 
         for key in IGNORED_FIELDS:
             super_dict.pop(key, None)
         return super_dict
 
+    # For now, set_widget_examples is not compatible with our transform/map operations, so we make it a no-op until it is fixed
```

Comment:
You can add this comment to the docstring of the function instead of putting it before